### PR TITLE
Collections/Message: add close button

### PR DESCRIPTION
**Pull request by domenkozar**

### CODE REVIEW

File: SemanticUI/Collections/Message.elm [hunk 1]
```diff
@@ -108,45 +127,46 @@ view extraAttributes cfg contents =
         )
         (case cfg.icon of
             Nothing ->
-                contentWithHeader
+                closeContent ++ allContent
 
             Just a ->
+                closeContent ++
```
Comment (ocharles) [inline SemanticUI/Collections/Message.elm]: This is in both branches so I think we should move it outside the `case` statement.

**Review by ocharles — CHANGES_REQUESTED** [review]

File: SemanticUI/Collections/Message.elm [hunk 2]
```diff
@@ -41,54 +42,72 @@ A message can contain an icon.
...
+view : List (Attribute msg) -> Config msg -> List (Html msg) -> Html msg
 view extraAttributes cfg contents =
     let
-        contentWithHeader =
+        closeContent =
+            case cfg.close of
+              Nothing ->
+                  []
+              Just msg ->
+                  [ i [ class "close icon"
```
Comment (ocharles) [inline SemanticUI/Collections/Message.elm]: Could we use `SemanticUI.Elements.Icon` here?

**Comment by domenkozar** [comment]
I think I've addressed the review, thanks Ollie :)

**Comment by ocharles** [comment]
Thanks!

**Pull request merged**